curated_transformers/generation/stop.py

```diff
@@ -0,0 +1,100 @@
+from abc import ABC, abstractmethod
```

Comment:
Why not call the module `stop_conditions.py`?

Reply:
Good idea, fixed.